src/tqec/block/enums.py

```diff
@@ -20,7 +20,7 @@ def to_template_sides(self) -> list[TemplateSide]:
                 TemplateSide.RIGHT,
                 TemplateSide.BOTTOM_RIGHT,
             ]
-        elif self == BlockDimension.Y:
+        if self == BlockDimension.Y:
```

Comment:
Not sure about the necessity of changing that to `if`. I think `elif` better shows the programmer intent, but `if` also works.

Reply:
It's a suggested change by pylint.